sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/MultiPathImporter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php.phpunit;
+
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.batch.sensor.SensorContext;
+
+public class MultiPathImporter implements ReportImporter {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MultiPathImporter.class);
+
+  private final SingleFileReportImporter importer;
+  private final String pathsKey;
+  private final String msg;
+
+  public MultiPathImporter(SingleFileReportImporter importer, String pathsKey, String msg) {
+    this.importer = importer;
+    this.pathsKey = pathsKey;
+    this.msg = msg;
+  }
+
+  @Override
+  public void importReport(SensorContext context, Map<String, Integer> numberOfLinesOfCode) {
+    final String[] paths = context.settings().getStringArray(pathsKey);
+    if (paths == null) {
+      LOGGER.info("No PHPUnit " + msg + " reports provided (see '" + pathsKey + "' property)");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2629)